### ISSUE TITLE
windterm exit unexpectedly

### ISSUE THREAD

**Issue by zangetsudo** [kingToolbox/WindTerm#436]
windterm exit  after the serial com continue output with a lot of data, say text.
we dont know when it exit,but it just annoying us a lot.

**Comment by kingToolbox** [comment]
Sorry, the instability of `WindTerm` has caused you trouble.

If possible, Would you like to help me reproduce it once and submit the generated crash report here? The crash report is just in the current working directory after the crash, such as `8f9a5ae5-4ea2-170e-55e9b70e-05c9ab40.dmp`, you can post it as an attachment in the issue. I will analyze it and fix the problem. Thank you.

The crash report is a standard `minidump` file and does not contain any private information. You can read [this comment](https://github.com/kingToolbox/WindTerm/issues/110#issuecomment-875854346) to learn more.

**Comment by kingToolbox** [comment]
Thank you very much, I will check the report and let you know the result.

**Comment by zangetsudo** [comment]
any updates?

**Comment by kingToolbox** [comment]
Yes, I have tried to fix it, please download the [WindTerm_2.3.0_Prerelease_2](https://github.com/kingToolbox/WindTerm/releases/tag/2.3-prerelease) version and check it.

I have been testing printing large files for the past two days and have not encountered a crash. but I do not have a real serial device, so I am not sure if this bug has been completely fixed. If you still encounter problems or even crashes, please let me know and I will continue to fix it, thank you.

**Comment by JSylvainGranier** [comment]
Hi, 
I'm experiencing same problem with SSH connections to linux hosts. 

When I try to open or download file above a specific size, WindTerm crashes. 

- Up to 500 Kb is OK. 👍 
- 17 000 Kb is still OK 👍 
- 42 000 Kb is KO 💥 

I test with logs files that WinSCP can download without any issues. 

The file is created in the target folder on my computer, but the file is empty. 

WindTerm 2.3.1 portable 64b on Windows 10 Pro 21H2 64b

I tryed a fresh install with all new profil directory : same result. 

And i can't find the dmp file 😭
